x/ccv/provider/keeper/grpc_query.go

```diff
@@ -139,9 +139,12 @@ func (k Keeper) GetConsumerChain(ctx sdk.Context, consumerId string) (types.Chai
 		strDenylist[i] = addr.String()
 	}
 
+	// TODO PERMISSIONLESS: metadata is deleted on chain stop.
+	// just log and error and accept empty content for now
+	// to make e2e test happy
 	metadata, err := k.GetConsumerMetadata(ctx, consumerId)
-	if err != nil {
-		return types.Chain{}, fmt.Errorf("cannot get metadata for consumer (%s): %w", consumerId, err)
+	if err != nil && phase != types.ConsumerPhase_CONSUMER_PHASE_STOPPED {
```

Comment:
If the chain is stopped probably it doesn't make sense to retrieve any chain data as it was deleted. This should be checked when getting the phase.

Reply:
I think it makes sense to keep consumer metadata, initialization and power-shaping parameters, even after a chain has stopped. (Verified this with @MSalopek).

Reply:
well I was not sure if there's a use case for post-mortem analysis where this command could be useful.